mkdocs.yml

```diff
@@ -134,10 +134,11 @@ nav:
       - 'Temporary Voting for Governance': 'governance/temporary-voting.md'
 
 - 'Design docs':
-    - 'Roadmap': "roadmap.md"
     - 'git-submodules': 'design/git-submodules.md'
     - 'git-submodule-storage': 'design/git-submodule-storage.md'
     - 'JJ run': 'design/run.md'
     - 'Sparse Patterns v2': 'design/sparse-v2.md'
     - 'Tracking branches': 'design/tracking-branches.md'
     - 'Copy tracking and tracing': 'design/copy-tracking.md'
+
+- Roadmap: "roadmap.md"
```

Comment:
I was thinking it should be under Development since it's a Development Roadmap (as opposed to, say, roadmap for learning `jj`).

If you think it's better on the top-level, I'd call it "Development Roadmap" there.

Reply:
done